ocp:
    dt: 0.01
    horizon_size: 20
    armature: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
    gripper_weight: 500
    state_weight: 5
    control_weight: 0.001
    max_qp_iter: 100
    max_iter: 7
    use_constraints: True
    activate_callback: False
    effector_frame_name: "panda_hand_tcp"
save_predictions_and_refs: True
rate: 100
use_vision: False
use_vision_simulated: True
start_visual_servoing_dist: 0.08
increasing_weights:
    max: 100
    percent: 0.9
    time_reach_percent: 6
